Filters/exceptions.txt

```diff
@@ -18,18 +18,8 @@
 @@||oneline.nextday.media^|
 ! Reported via Telegram: https://t.me/AdGuard_chinese/13741
 @@|ftp.bmp.ovh^|
-!
-! Related to '||mkto-*.com^$third-party'
-! https://github.com/AdguardTeam/AdGuardSDNSFilter/issues/641
-@@||click.acloud.guru^|
-! https://github.com/AdguardTeam/AdGuardSDNSFilter/issues/627
-@@||email.questrade.com^|
-! https://github.com/AdguardTeam/AdGuardSDNSFilter/issues/582
-@@||email.awscloud.com^|
-!
 ! https://github.com/AdguardTeam/AdGuardSDNSFilter/issues/510
 @@||c.digikey.com^|
```

Comment:
Maybe this exceptions should be also removed.
If I'm not wrong, it was also blocked by `||mkto-*.com^` as CNAME.

Reply:
Removed